Formula/terminator.rb

```diff
@@ -0,0 +1,46 @@
+class Terminator < Formula
+  include Language::Python::Virtualenv
+
+  desc "Multiple GNOME terminals in one window"
+  homepage "https://gnome-terminator.org"
+  url "https://github.com/gnome-terminator/terminator/archive/refs/tags/v2.1.1.tar.gz"
+  sha256 "ee1907bc9bfe03244f6d8074b214ef1638a964b38e21ca2ad4cca993d0c1d31e"
+  license "GPL-2.0-only"
+
+  depends_on "dbus"
+  depends_on "pygobject3"
+  depends_on "python@3.9"
+  depends_on "vte3"
+
+  resource "dbus-python" do
+    url "https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz"
+    sha256 "11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4"
+  end
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/e1/b0/7276de53321c12981717490516b7e612364f2cb372ee8901bd4a66a000d7/psutil-5.8.0.tar.gz"
+    sha256 "0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6"
+  end
+
+  resource "configobj" do
+    url "https://files.pythonhosted.org/packages/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz"
+    sha256 "a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    pid = spawn("DYLD_FALLBACK_LIBRARY_PATH=#{HOMEBREW_PREFIX}/lib #{bin}/terminator -d > #{testpath}/terminator.out 2>&1")
```

Comment:
Needing to set `DYLD_FALLBACK_LIBRARY_PATH` is a bit worrying. What happens when you don't have this? If I `brew install terminator`, will I need to set this for `terminator` to work too?

Also, we should try avoiding shell redirection with `>`, as this often causes deadlocks in CI. (Something about orphan processes/pipes that `brew` isn't able to spot as needing to be killed when the test times out...)

The docs should show an idiomatic way for setting env variables and redirecting stdout/stderr. https://ruby-doc.org/core-2.6.3/Process.html#method-c-spawn

Reply:
Yeah, the need to set DYLD_FALLBACK_LIBRARY_PATH is caused by the fact that I'm not actually running homebrew from /usr/local.  It works fine when the homebrew libs are where they are expected, but when the libs are not in /usr/local/lib then the brew test gets confused.  I was thinking about printing something to stdout after install if I detect that HOMEBREW_PREFIX is not /usr/local.  Is this a good idea.

I was just reading the docs for Process.spawn, and I'll fix this up in the next go.  

Thank you for all your help.  It's been very helpful.  I'll take a look at neovim-remote as well.

Reply:
We're going to need to fix this so that it doesn't need `DYLD_FALLBACK_LIBRARY_PATH`, I'm afraid. Needing to set it doesn't play well with SIP.

Reply:
That's easy to do, it was mostly for my convenience anyways.

Reply:
Hmm, well, even though the test now passes with homebrew installed in /usr/local, it seems to fail in the CI test harness with the same error message as it did when I had when I had homebrew installed in my home directory.

Reply:
It seems like python or pygobject3 can't find 'libpango-1.0.0.dylib' if it's not in /usr/local/lib

Reply:
The primary error seems to be this:
```
Failed to load shared library 'libpango-1.0.0.dylib' referenced by the typelib: dlopen(libpango-1.0.0.dylib, 9): image not found
```

#76383 should fix it. I'll go merge that now. You'll need to rebase to pull in the changes from there when that happens.

Reply:
Awesome!  Thanks for that.  Can I just pull that in with a `brew update`?

Reply:
also, is there a way to re-run the tests with the new patch to gobject-introspection when it gets merged?

Reply:
Merged. Please rebase with `git rebase` to avoid creating a merge commit on your PR branch.

Reply:
Oops, your last two comments didn't load for me when I posted my comment for some reason. Yes, just do
```
brew update
git rebase origin/master
git push --force
```
This will re-run the tests with the updated version of gobject-introspection.

Reply:
Actually, I can rebase this for you. I'll go ahead and do that.